src/ImpactX.cpp

```diff
@@ -199,6 +208,100 @@ namespace impactx {
                                        << " slice_step=" << slice_step << "\n";
                     }
 
+                    // CSR Wakefield Response
+
+                    bool element_has_csr = false; // Updates to true for example with bend element
+                    double R = 0.0; // Updates for bend element rc
+
+                    //Define lambda function inside of std::visit
+                    std::visit([&R, &element_has_csr](auto &&element)
+                    {
+                        if constexpr (std::is_same_v<std::decay_t<decltype(element)>, Sbend>)
+                        {
+                            R = element.m_rc;
+                            std::cout << "My radius of curvature is:" << R << std::endl;
+                            element_has_csr = true;
+                        }
+                        else if constexpr (std::is_same_v<std::decay_t<decltype(element)>, CFbend>)
+                        {
+                            R = element.m_rc;
+                            std::cout << "My radius of curvature is:" << R << std::endl;
+                            element_has_csr = true;
+                        }
+                        /*
+                        else if constexpr (std::is_same_v<std::decay_t<decltype(element)>, ExactSbend>) //Currently internal calculation for m_rc
+                        {
+                            R = element.m_rc;
```

Comment:
This case (the `ExactSBend`) will cause a problem because `m_rc` is not a member variable.  Instead, `rc` is computed internally from other parameters.  For now, we could add a comment that this case will be addressed later.  We can make a simple change to ExactSBend to address this.

Reply:
Adding a TODO now in the code.

Reply:
Actually, I can implement it real quick.